cdap-common/src/main/java/co/cask/cdap/common/lang/ClassLoaders.java

```diff
@@ -79,4 +83,20 @@ public static ClassLoader setContextClassLoader(ClassLoader classLoader) {
     // The casting should succeed since it's either null or is assignable to the given type
     return (T) result;
   }
+
+  /**
+   * Populates the list of {@link URL} that this ClassLoader uses, including all URLs used by the parent of the
+   * given ClassLoader.
+   */
+  public static List<URL> getClassLoaderURLs(ClassLoader classLoader, List<URL> urls) {
```

Comment:
I think `get` is fine since it is getting the `URL`s from the given ClassLoader. The collection is for storing the result.